playbooks/roles/airship-deploy-osh/tasks/main.yml

```diff
@@ -134,7 +137,8 @@
     SHIPYARD_IMAGE: "{{ shipyard_image }}"
     OS_PASSWORD: "{{ lookup('password', secrets_location + '/ucp_shipyard_keystone_password ' + password_opts) }}"
   register: shipyard_update_software
-  failed_when: shipyard_update_software.stdout.find('Error') == 0
+  failed_when:
+    - shipyard_update_software.stdout.find('Errors: 0') != 0
```

Comment:
you should wrap this with double quotes like above.

Reply:
argh

Reply:
and probably add a different condition :)